File: Day-34/Purvi.py
class Solution {
public:
    int maximum69Number (int num) {
        int i=0, j, d=num, l, a[100];
        stringstream ss;
        while(d!=0)
        {
            a[i]=d%10;
            d/=10;
            i++;
        }
        for(j=i-1; j>=0; j--)
        {
            if(a[j]==9)
                continue;
            else
            {
                a[j]=9;
                break;
            }
        }
        for(j=i-1; j>=0; j--)
            ss<<a[j];
        ss>>l;
        return l;
    }
};

File: Day-52/Purvi.cpp
class Solution {
public:
    int reverse(int x) {
        int flag=1;
        long long rev=0, num, rem;
        if(x<0)
            flag = -1;
        num=llabs(x);
     
       while(num!=0)
        {
            rem=num%10;
            rev=rev*10+rem;
            num=num/10;
        }
        if(rev>2147483648)
            return 0;
        return flag*rev;
    }
};